### DISCUSSION TITLE
How can I implement { % trans % } and { % commit % }?

### DISCUSSION THREAD

**Discussion by boludoz** [pallets/jinja#2044] [General]
Is there any way to enable this?

**Comment by mesr** [comment]
Hello there,

I see this is an older question. Is it still current?

If so, would you mind explaining a bit more what are those two tags that you are referring to? They don't sound familiar to me, so I'm kind of assuming that they must be custom tags that you wish to implement.

I am still learning about Jinja2's extension API, but I have nonetheless already implemented a couple of custom tag of my own. Perhaps I can help.
